schemas/lifeCycleData.json

```diff
@@ -0,0 +1,214 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/building-envelope-data/api/v1.0.0/schemas/lifeCycleData.json",
+  "title": "life cycle data",
+  "description": "##",
+  "$defs": {
+    "data": {
+      "type": "object",
+      "title": "The object is used to define calorimetric data.",
+      "properties": {
+        "functionalUnit": {
+          "type": "string",
+          "enum": [
+            "Volume",
+            "Quantity",
+            "Area",
+            "Length",
+            "OtherUnit",
+            "Energy",
+            "Weight"
+          ],
+          "description": "Defines the type of product."
+        },
+        "defaultFunctionalUnitAmount": {
+          "$ref": "number.json#/$defs/nonNegativeNumber",
+          "description": "Describes the default functional unit value. For a quantity/volume product, this describes the default volume (m3), for area product the default area (m2), length product the default length (m) and for weight product the default weight (kg)"
+        },
+        "fixedHeight": {
+          "$ref": "number.json#/$defs/meter",
+          "description": "This value describes the fixed height (m) of an area product"
+        },
+        "fixedArea": {
+          "$ref": "number.json#/$defs/squareMeter",
+          "description": "This values describes the fixed area (m2) of a length product"
+        },
+        "density": {
+          "$ref": "number.json#/$defs/nonNegativeNumber",
+          "description": "The product density in kg/[functional unit]."
+        },
+        "weight": {
+          "$ref": "number.json#/$defs/kilogram",
+          "description": "The weight in kg for a quantity product."
+        },
+        "lifespan": {
```

Comment:
@simon-wacker I think it is ok to have "lifespan" in the domain "life-cycle data", because there may be several plausible assumptions. Do you agree?